src/elfpy/pricing_models/yieldspace.py

```diff
@@ -353,12 +353,12 @@ def calc_in_given_out(
         # We precompute the YieldSpace constant k using the current reserves and
         # share price:
         #
-        # k = (c / μ) * (μ * z)**(1 - t) + (2y + cz)**(1 - t)
+        # k = (c / μ) * (μ * z)**(1 - tau) + (2y + cz)**(1 - tau)
```

Comment:
You might as well use the unicode symbol for tau since we already do this for mu.

Reply:
I made this change, but it was cumbersome and I can see grepping etc being cumbersome in the future.  If you have no strong feelings I could to a search and replace all to get rid of greek symbols with alphabet spellings instead.

Reply:
in a follow up PR i mean

Reply:
agreed, I'd rather us spell it out (`mu`, `tau`, etc)

Reply:
I prefer $\mu$ and $\tau$, but I don't care enough to argue about it.